Fireworks/Core/src/FWGeometry.cc

```diff
@@ -10,6 +10,10 @@
 #include "Fireworks/Core/interface/fwLog.h"
 #include "DataFormats/DetId/interface/DetId.h"
 
+
+// AMT deprication of tracker specific DetIds
+#include "CalibTracker/StandaloneTrackerTopology/interface/StandaloneTrackerTopology.h"
```

Comment:
hi @alja - does fireworks already depend on this package? Its naively not part of fwlite @smuzaffar

Reply:
@davidlange6 , no CalibTracker/StandaloneTrackerTopology is not part of fwlite build set. If @alja is happy with changes in Fireworks/Core/BuildFile.xml then we will update fwlite application set to include CalibTracker/StandaloneTrackerTopology